lib/edit-site-page.php

```diff
@@ -198,9 +198,24 @@ function gutenberg_edit_site_init( $hook ) {
 	wp_enqueue_script( 'wp-format-library' );
 	wp_enqueue_style( 'wp-edit-site' );
 	wp_enqueue_style( 'wp-format-library' );
+
+	wp_deregister_script( 'editor' );
+	wp_deregister_script( 'wp-tinymce' );
+	wp_deregister_script( 'wp-tinymce-root' );
+	wp_deregister_script( 'wp-tinymce-lists' );
+	wp_deregister_script( 'mce-view' );
 }
 add_action( 'admin_enqueue_scripts', 'gutenberg_edit_site_init' );
 
+/**
+ * Disallow rich editing in the site editor.
+ * This prevents tinymce from loading.
+ */
+function gutenberg_disable_richedit_tinymce() {
+	return ! gutenberg_is_edit_site_page( get_current_screen()->id );
```

Comment:
What's the difference with the above? What will this do?

Reply:
Some TinyMCE inline scripts can't be removed by deregistering the scripts. But they are tied to this filter so by returning false here we can prevent them from loading.